### DISCUSSION TITLE
Canal Ecuavisa no funciona / Actualización

### DISCUSSION THREAD

**Discussion by jonathanguartan** [iptv-org/community#438] [Q&A] [answered]
Saludos. 

En enlace actualmente de este canal se encuentra caído.
Pero, funciona correctamente a través del siguiente enlace:
https://mdstrm.com/live-stream-playlist/603ea23f928cc5255563012c.m3u8

**Comment by BellezaEmporium** [comment]
Con ese enlace tengo un error "BLACKOUT" en quién se refere a un otro enlace de Mediastream.

Pero este enlace se bloquea con un error referiendo a la localizacion.

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream
